sdk/keyvault/azure-security-keyvault-keys/src/private/key_constants.hpp

```diff
@@ -13,48 +13,17 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Keys { nam
   constexpr static const char KeyVaultServicePackageName[] = "keyvault-keys";
 
   /***************** KeyVault Key *****************/
-  constexpr static const char KeyPropertyName[] = "key";
+  // constexpr static const char KeyPropertyName[] = "key";
 
   /***************** Key Client *****************/
   constexpr static const char KeysPath[] = "keys";
-  constexpr static const char DeletedKeysPath[] = "deletedkeys";
+  // constexpr static const char DeletedKeysPath[] = "deletedkeys";
 
   /***************** Key Properties *****************/
-  constexpr static const char ManagedPropertyName[] = "managed";
-  constexpr static const char AttributesPropertyName[] = "attributes";
   constexpr static const char TagsPropertyName[] = "tags";
-  constexpr static const char ReleasePolicyPropertyName[] = "release_policy";
-  constexpr static const char EnabledPropertyName[] = "enabled";
-  constexpr static const char ExportablePropertyName[] = "exportable";
-  constexpr static const char NbfPropertyName[] = "nbf";
-  constexpr static const char ExpPropertyName[] = "exp";
-  constexpr static const char CreatedPropertyName[] = "created";
-  constexpr static const char UpdatedPropertyName[] = "updated";
-  constexpr static const char RecoverableDaysPropertyName[] = "recoverableDays";
-  constexpr static const char RecoveryLevelPropertyName[] = "recoveryLevel";
-  constexpr static const char HsmPlatformPropertyName[] = "hsmPlatform";
-
-  /***************** Key Request Parameters *****************/
-  constexpr static const char KeyTypePropertyName[] = "kty";
-  constexpr static const char KeySizePropertyName[] = "key_size";
-  constexpr static const char KeyOpsPropertyName[] = "key_ops";
-  constexpr static const char CurveNamePropertyName[] = "crv";
-  constexpr static const char PublicExponentPropertyName[] = "public_exponent";
```

Comment:
Do these all go away because of the serializer cleanup?

Reply:
yes, and because they are defined in the generated code